omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -270,9 +271,20 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 			}
 			
 			new UserProperties(user.getUserProperties()).setSupposedToChangePassword(forcePassword);
-			
 			userValidator.validate(user, errors);
-			
+			if (errors.hasErrors()) {
+				List<FieldError> fieldErrorList = errors.getFieldErrors();
+				for (FieldError fieldError : fieldErrorList) {
+					String[] errorCodes = fieldError.getCodes();
+					for (String value : errorCodes) {
+						if (value.contains("error.exceededMaxLengthOfField")) {
+							httpSession.setAttribute(WebConstants.OPENMRS_ERROR_ATTR, "User.DemographicInfo.length");
```

Comment:
thanks @Parth59 ,some times sessions get disturbing more so when they are not destroyed after use,why not use model .addAttribute( )

Reply:
Hi @HerbertYiga , I have started reading about the same. Will try to implement those changes in the code !

Reply:
Hi @HerbertYiga

I tried using the model.addAttrribute() and it seems like using those with redirect causes them to be added to the URL. Additionally, the <c:if> tag in the view doesn't load up the error messages as can be seen in the screenshot referenced below.  [isLengthExceeded in the url comes to be true but isn't loaded in the page]

Infact, I also double checked for places that utilize model.addAttribute() in the same function. Turns out that the entire function utilizes httpSession for passing response back to the user. 
So requesting you to kindly let this security PR pass . If needed I raise an issue and then will refactor to modify the entire function in a separate PR. Sounds good ? 

![ScreenShot](https://user-images.githubusercontent.com/22288099/125032981-97414580-e043-11eb-84bd-6f9f8614d460.png)